### PR TITLE
Fixed unresponsive table on admin/user web page, development env setup without vagrant, setup sql erroe and secret key addition.

### PR DESCRIPTION
**Pull request by awasum**
fixed secret key generation error
renamed config files from example.
Fixed unresponsive table on admin/user web page, development env setup without vagrant, setup sql erroe and secret key addition.

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@
 Devise.setup do |config|
   # ==> Secret key, generate one with `rake secret`
   config.secret_key = Rails.application.secrets.devise_secret_key
+  config.secret_key = '0014809cd45aa9fc57a29118bbbc374437095fed0b39549b69aef61b9ad994f0e71a0bf54e66a5acec3d67d293495c1b6f4235e8b6415e4fe2bafb8f517121ea'
```
Comment (sonalkr132) [inline config/initializers/devise.rb]: Please remove this.

Comment (sonalkr132) [inline config/initializers/devise.rb]: @awasum You can set `devise_secret_key` in `secret.yml` file.

Comment (awasum) [inline config/initializers/devise.rb]: That true @sonalkr132, i had a problem when setting up my development environment.

Comment (awasum) [inline config/initializers/devise.rb]: will remove it

**Comment by sonalkr132** [comment]
Please [squash](http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git) your commits as well. You don't need two commits.

**Comment by ChrisBr** [comment]
thanks @awasum for your contribution, however, the issue is already resolved! But keep up the great work

Closing ...

**Pull request closed without merging**